src/main/kotlin/org/rust/lang/core/resolve/RustResolveEngine.kt

```diff
@@ -150,6 +155,19 @@ object RustResolveEngine {
         return resolveIn(ResolveScopeVisitor(ref, visited), enumerateScopesFor(ref))
     }
 
+    private fun resolveImportSpec(ref: RustImportSpec, visited: MutableSet<RustUseItem>): ResolveResult {
+        val basePath = ref.parentOfType<RustUseItem>()?.viewPath?.pathPart ?: return ResolveResult.UNRESOLVED
+
+        // NOTE(matklad): this is not necessary a module
```

Comment:
Or do you think that such notes are just clutter in code? 

Or may be it's better to use Rust code snippets for such cases?